janus/lib/client/jsx/__tests__/janus-settings.test.jsx

```diff
@@ -11,18 +11,26 @@ describe('JanusSettings', () => {
   beforeEach(() => {
     store = mockStore({
       user: {
-	email: "janus@two-faces.org",
-	name: "Janus Bifrons",
-	permissions: { }
+        email: "janus@two-faces.org",
+        name: "Janus Bifrons",
+        permissions: { }
       }
     });
   });
 
+  const createNodeMock = element => {
```

Comment:
This is a workaround for a known issue with MUI multiline `TextField` for testing...

https://github.com/mui/material-ui/issues/16491